### PR TITLE
[OCaml] Fixup after #280

### PR DESCRIPTION
**Pull request by nbacquey**
Fixes double bracketing and update comment in the sample file.

### CODE REVIEW

**Review by Xophmeister — APPROVED** [review]
Sorry! I completely misunderstood what you meant by double-bracketing

File: tests/samples/expected/ocaml.ml [hunk 1]
```diff
@@ -931,7 +931,7 @@ let x =
     | true -> 1.
   ]
 
-(* New line for structures in module definitions if they appear in the input *)
+(* New line for structures in module definitions *)
```
Comment (aspiwack) [inline tests/samples/expected/ocaml.ml:934]: Does this diff imply that if I write a one-line module a newline will be inserted and it'll become multiline?

Comment (Xophmeister) [reply tests/samples/expected/ocaml.ml:934]: It's still a softline, so a one-line module will remain on a single line.

Comment (nbacquey) [reply tests/samples/expected/ocaml.ml:934]: The diff means that a newline will be inserted if the context is multi-line, even if the newline isn't _precisely there_ in the input (basically it's `scoped_softline` versus `input_softline`). Your one-line module would stay as such.

**Pull request merged**